MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,224 @@
+
+If the validation mode is LostFocus, the value will be validated when the editor lost its focus. By default the [ValidationMode] is `LostFocus`
+
+### PropertyChanged
+
+If the validation mode is PropertyChanged, The value will be validated immediately when it is changed.
+
+### Manual
+
+The value should be validated manually by calling the [SfDataForm.Validate] or [SfDataForm.Validate(new List<string>())] method.
+
+The following code validates the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight c# %}
+
+this.DataForm.Validate();
+
+{% endhighlight %}
+{% endtabs %}
+
+To validate the specific list of property value, pass the property names as argument.
+
+{% tabs %}
+{% highlight c# %}
+this.DataForm.Validate(new List<string>() {"FirstName", "Age" });
+{% endhighlight %}
+{% endtabs %}
+
+You can determine whether the data form or property is valid or not by using the `Validate` method.
+
+{% tabs %}
+{% highlight c# %}
+
+        bool isValid = DataForm.Validate();
+        bool isPropertyValid = DataForm.Validate(new List<string>());
+
+{% endhighlight %}
+{% endtabs %}
+
+If the data form or property is valid, `true` will be returned. Or else `false` will be returned.
+
+### Get Validation details through events
+
+You can validate the entire data form using the [ValidateForm] event of the data form.
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        ValidateForm="OnDataFormValidateForm">
+    </DataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.DataForm.ValidateForm += this.OnDataFormValidateForm;
+
+    private void OnDataFormValidateForm(object sender, DataFormValidateFormEventArgs e)
+    {
+        object dataObject = e.DataObject;
+        var errorMessage = e.ErrorMessage;
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Valid message
+
+If the value meets the desired criteria, you can show the [ValidMessage]. As error message, the valid message will also be displayed at the bottom of the editor.
+
+{% tabs %}
+{% highlight c# %}
+
+    [DataFormDisplayOptions(ColumnSpan = 2, ValidMessage = "Password strength is good")]
+    [Required(ErrorMessage = "Please enter the password")]
+    [RegularExpression(@"^(?=.*[a-z])(?=.*[A-Z])[a-zA-Z\d]{8,}$", ErrorMessage = "A minimum 8-character password should contain a combination of uppercase and lowercase letters.")]
+    public string Password { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+The [ValidMessageLabelTextStyle] property of [SfDataForm] and [DataFormItem] class is used to customize the text color, font, font size, font family and font  style of valid message label.
```

Comment:
### Customize error label text style
The error label style can be customized by changing the [ErrorLabelTextStyle] property of [SfDataForm].

Code
 <dataForm:SfDataForm x:Name="dataForm" >
            <dataForm:SfDataForm.ErrorLabelTextStyle>
                <dataForm:DataFormTextStyle FontSize="10" FontAttributes="Italic" TextColor="Violet" FontFamily="Roboto"/>
            </dataForm:SfDataForm.ErrorLabelTextStyle>
        </dataForm:SfDataForm>

We can also customize error label style for each editor using [ErrorLabelTextStyle] property of [DataFormItem]

Code
  private void OnGenerateDataFormItem(object sender, Syncfusion.Maui.DataForm.GenerateDataFormItemEventArgs e)
    {
        if (e.DataFormItem != null)
        {
            if (e.DataFormItem.FieldName == nameof(EditorsModel.FirstName))
{
                textItem.ErrorLabelTextStyle = new DataFormTextStyle
                {
                    TextColor = Colors.DarkSeaGreen,
                    FontSize = 8,
                    FontAttributes = FontAttributes.Italic,
FontFamily = "Roboto",
                };
}

Reply:
Add like above for Error label style and validation label style

Reply:
changes added to  Error label style and validation label style